examples/C++/HelloWorldExampleTCP/CMakeLists.txt

```diff
@@ -48,9 +48,13 @@ file(GLOB HELLOWORLD_EXAMPLE_SOURCES_CPP "*.cpp")
 #message(STATUS "XML Files: " ${XML_CONFIG_FILES})
 configure_file("HelloWorldSubscriber.xml" "HelloWorldSubscriber.xml" COPYONLY)
 configure_file("HelloWorldPublisher.xml" "HelloWorldPublisher.xml" COPYONLY)
+if(TLS_FOUND)
```

Comment:
If this example is compiled outside FastRTPS, these files won't be copied. Should be better to remove the condition and always copy these files.